controller_app/settings.py

```diff
@@ -21,12 +22,16 @@
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-7x$$d9^(-rb2uj57*d%u#z68fh*@y8s)8zu(*_t-ck%5jmk)ef"
+# TODO add environment variables to dokku app
+# TODO Note: these env vars are labelled with the additional "JOBRUNNER" prefix
```

Comment:
How about CONTROLLER rather than JOBRUNNER? I think that maybe clearer as to ownership, and I wonder if we want to move away from the term job-runner in general?

Reply:
Sure